recipes/rsem/build.sh

```diff
@@ -34,6 +37,55 @@ rsem-tbam2gbam \
 rsem_perl_utils.pm \
 "
 
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+
+########################################################
+### Prepare Samtools/Htslib 
+########################################################
+
+sed -i.bak 's/^CPPFLAGS =$//g' samtools-1.3/Makefile
+sed -i.bak 's/^LDFLAGS  =$//g' samtools-1.3/Makefile
+
+sed -i.bak 's/^CPPFLAGS =$//g' samtools-1.3/htslib-1.3/Makefile
+sed -i.bak 's/^LDFLAGS  =$//g' samtools-1.3/htslib-1.3/Makefile
+
+# varfilter.py in install fails because we don't install Python
+sed -i.bak 's#misc/varfilter.py##g' samtools-1.3/Makefile
+
+# Remove rdynamic which can cause build issues on OSX
+# https://sourceforge.net/p/samtools/mailman/message/34699333/
+sed -i.bak 's/ -rdynamic//g' samtools-1.3/Makefile
+sed -i.bak 's/ -rdynamic//g' samtools-1.3/htslib-1.3/configure
+
+########################################################
+### Build htslib 
+########################################################
+cd samtools-1.3/htslib-1.3
```

Comment:
Same here, the libs and includes should be in your $PREFIX as you depend on it, already.